src/fontra/backends/designspace.py

```diff
@@ -519,6 +522,9 @@ async def putGlyph(
                 storeInLib(layerGlyph, GLYPH_DESIGNSPACE_LIB_KEY, localDS)
                 storeInLib(layerGlyph, SOURCE_NAME_MAPPING_LIB_KEY, sourceNameMapping)
                 storeInLib(layerGlyph, LAYER_NAME_MAPPING_LIB_KEY, layerNameMapping)
+                if GLYPH_NOTE_LIB_KEY in glyph.customData:
```

Comment:
It's not really specified here:
https://unifiedfontobject.org/versions/ufo3/glyphs/glif/#note
Why I changed it to an empty string as this is what I have done for `class UFOGlyph` 
Or should I use`note: str = None` instead of `note: str = ""` within `class UFOGlyph`?

Reply:
Can you try it out, or find the relevant code in ufoLib?

Reply:
mmhhh, I was hoping to find something relevant, but I was not able to do so. I found this:
https://github.com/unified-font-object/ufoLib/blob/e09067df19dd5d6f47c838fd0b1b19958cc8d7c3/Lib/ufoLib/glifLib.py#L330
But it does not tell me anything about the default. 

With fonttools defcon I was able to get a 'None':
<img width="1053" alt="Screenshot 2024-06-28 at 15 49 48" src="https://github.com/googlefonts/fontra/assets/21055547/4c8daa61-3c6b-4a79-8d26-23f4b30bfcec">

Why I changed it to None, now.

Reply:
The repo you refer to has been archived in 2019 (it says so loudly at the top of the page you link to). ufoLib has been part of fonttools since then, as you can tell from the imports in Fontra. The defcon library is completely irrelevant as we don't use it.

The relevant code is here, from which we deduce it doesn't matter whether we use `None` or `""`:

https://github.com/fonttools/fonttools/blob/221c518917c1430390b52c964c2fb0c752fb391f/Lib/fontTools/ufoLib/glifLib.py#L775-L776